.github/workflows/spack_build.yml

```diff
@@ -99,13 +99,16 @@ jobs:
     strategy:
       matrix:
         spack_spec:
-          - hiop@develop+mpi~raja~shared~kron~sparse ^openblas ^openmpi ^libevent~openssl
-          - hiop@develop~mpi~raja~shared~kron~sparse ^openblas ^libevent~openssl
-          - hiop@develop~mpi+raja~shared~kron~sparse ^openblas ^libevent~openssl
-
-            # We will need coinhsl for this, but what are the rules for using
-            # a coinhsl tarball?
-            # - hiop@develop~mpi~raja~shared~kron+sparse
+          - hiop@develop+mpi~raja~shared~kron~sparse ^openmpi 
+          - hiop@develop~mpi~raja~shared~kron~sparse
+          - hiop@develop~mpi+raja~shared~kron~sparse
+          # We will need coinhsl for this, but what are the rules for using
+          # a coinhsl tarball?
+          # - hiop@develop~mpi~raja~shared~kron+sparse
+        blas_provider:
+          - openblas
+        compiler:
+          - gcc
```

Comment:
Feel free to add other variants of your current build, or add compiler / blas specifications as wanted.

Would be easy enough to extend this to build / test with different MPI providers as well.